### ISSUE TITLE
Results for ogbg-molpcba

### ISSUE THREAD

**Issue by chaitjo** [lsj2408/GraphNorm#1]
Hi, thanks for open-sourcing the code and releasing extensive experiments! GNN-specific normalization beyond BatchNorm and LayerNorm was definitely a missing component in the toolbox. 

Did you experiment with GraphNorm on ogbg-molpcba (the larger and more challenging mol dataset from OGB) as well as the other graph classification tasks? Would love to know your experience or informal results.

**Comment by lsj2408** [comment]
Thanks for watching our work! 
Indeed, we are performing further experiments on the large-scale datasets as you mentioned (ogbg-molpcba and ogbg-ppa). Moreover, we will try more GNN architectures to better understand the power of GraphNorm.  The new results will be updated as soon as possible.

**Comment by chaitjo** [comment]
Awesome, keep me in the loop; looking forward to it.

**Comment by chaitjo** [comment]
If I want to use GraphNorm as a plug-and-play replacement for PyTorch's BatchNorm layer with DGL, which code snippet do you suggest? 
- https://github.com/lsj2408/GraphNorm/blob/master/GraphNorm_ws/ogbg_ws/src/dgl_model/norm.py
- https://github.com/lsj2408/GraphNorm/blob/master/GraphNorm_ws/gnn_ws/gnn_example/model/Norm/norm.py

**Comment by lsj2408** [comment]
Both are fine~ You could take the second one for your tasks. Just try them for free.

There is also a thrid-part implementation by [Microsoft ptgnn](https://github.com/microsoft/ptgnn):
- https://github.com/microsoft/ptgnn/blob/master/ptgnn/neuralmodels/gnn/messagepassing/graphnorm.py

**Comment by felipemello1** [comment]
@lsj2408, thanks for sharing this repo.

It seems that your code here is missing alpha: https://github.com/lsj2408/GraphNorm/blob/master/GraphNorm_ws/ogbg_ws/src/dgl_model/norm.py

![image](https://user-images.githubusercontent.com/23004953/96827756-44b3f080-1404-11eb-826c-dc078753e7a7.png)


Did I miss it?

**Comment by hugovergnes** [comment]
> @lsj2408, thanks for sharing this repo.
> 
> It seems that your code here is missing alpha: https://github.com/lsj2408/GraphNorm/blob/master/GraphNorm_ws/ogbg_ws/src/dgl_model/norm.py
> 
> ![image](https://user-images.githubusercontent.com/23004953/96827756-44b3f080-1404-11eb-826c-dc078753e7a7.png)
> 
> Did I miss it?

I think so too, looks like in the microsoft implementation above they added
